src/components/BottomNav.jsx

```diff
@@ -10,13 +10,13 @@ const BottomNav = ({ captureNavigation }) => {
 
   return (
       <div className="bottomnav">
-        <div className="bottomnavitem" name="explore" onClick={e => captureNavigation(e)}><img src={pantry} name="explore" onClick={e => captureNavigation(e)}></img></div>
+        <div className="bottomnavitem" name="explore" onClick={e => captureNavigation(e.target.getAttribute('name'))}><img src={pantry} name="explore" onClick={e => captureNavigation(e.target.getAttribute('name'))}></img></div>
```

Comment:
Since the name is preset anyway, could it not simply be `captureNavigation('explore')`?